### ISSUE TITLE
Dewikification: Migrate all existing guides on pythondiscord.com to the new guides system.

### ISSUE THREAD

**Issue by lemonsaurus** [python-discord/site#384]
**Remember to target the `dewikification` branch with this pull request**

This issue must be solved _after_ #383.

Once a new system for the guides is ready, we should migrate our existing guides into this new system.

Please make this a separate pull request, so that we can review the system and the content separately.
